src/TableView.elm

```diff
@@ -202,7 +204,7 @@ appendFirstRowCell dateWithSessions timeDelimiters model numColumns index column
             if String.isEmpty trackName then
                 "none"
             else
-                "block"
+                "inline-block"
```

Comment:
In testing this only seems to work for sessions in the first column / the first session (i'm not sure) but the little square is present in other sessions:
![screen shot 2017-03-16 at 13 32 49](https://cloud.githubusercontent.com/assets/7805691/23998458/191e079a-0a4d-11e7-8693-e589e0bb37ec.png)